newIDE/app/src/BehaviorsEditor/Editors/Physics2Editor/index.js

```diff
@@ -414,25 +426,29 @@ const Physics2Editor = (props: Props) => {
               vertices[index].x = newValue;
               behavior.updateProperty('vertices', JSON.stringify(vertices));
               forceUpdate();
+              onBehaviorsUpdated();
             }}
             onChangeVertexY={(newValue, index) => {
               let vertices = JSON.parse(properties.get('vertices').getValue());
               vertices[index].y = newValue;
               behavior.updateProperty('vertices', JSON.stringify(vertices));
               forceUpdate();
+              onBehaviorsUpdated();
             }}
             onAdd={() => {
               let vertices = JSON.parse(properties.get('vertices').getValue());
               if (vertices.length >= 8) return;
               vertices.push({ x: 0, y: 0 });
               behavior.updateProperty('vertices', JSON.stringify(vertices));
               forceUpdate();
+              onBehaviorsUpdated();
             }}
             onRemove={index => {
               let vertices = JSON.parse(properties.get('vertices').getValue());
               vertices.splice(index, 1);
               behavior.updateProperty('vertices', JSON.stringify(vertices));
               forceUpdate();
+              onBehaviorsUpdated();

```

Comment:
should we create a function:
```
const updateBehaviorProperty = (property, value) => {
  behavior.updateProperty(property, value);
  forceUpdate();
  onBehaviorsUpdated();
}
```

This would ensure we always call them in order and would simplify the component

Reply:
Good remark, would be worth it I guess.